File: 2st/Yeonwoo/특별한 이차원 배열 2.py
def solution(arr):
    answer = 0
    cnt = 0
    for i in range(len(arr)):
        for j in range(len(arr[0])):
            if arr[i][j]==arr[j][i]:
                cnt += 1
    if cnt==len(arr)*len(arr[0]):
        answer = 1
    return answer

File: 4st/Eunseong/특별한 이차원 배열 2.java
class Solution {
    public int solution(int[][] arr) {
        for (int i = 0; i < arr.length; i++) {
            for (int j = 0; j < arr[i].length; j++) {
                if (arr[i][j] != arr[j][i]) return 0;
            }
        }
        return 1;
    }
}